components/SocketBridge/SocketBridge.tsx

```diff
@@ -0,0 +1,113 @@
+import { Bridge } from '@socket.tech/plugin';
+import { useCallback } from 'react';
+import styled, { useTheme } from 'styled-components';
+import { chain } from 'wagmi';
+
+import ArrowIcon from 'assets/svg/app/arrow-down.svg';
+import Connector from 'containers/Connector';
+import { fetchBalances } from 'state/balances/actions';
+import { useAppDispatch } from 'state/hooks';
+
+import { SocketCustomizationProps } from './types';
+import hexToRGB, {
+	DEFAULT_MOBILE_WIDTH,
+	DEFAULT_WIDTH,
+	SOCKET_DEST_TOKEN_ADDRESS,
+	SOCKET_SOURCE_TOKEN_ADDRESS,
+} from './utils';
+
+const socketDefaultChains = [
+	chain.arbitrum.id,
+	chain.mainnet.id,
+	chain.optimism.id,
+	chain.polygon.id,
+];
+
+const SocketBridge = () => {
+	const { activeChain, signer } = Connector.useContainer();
+	const dispatch = useAppDispatch();
+	const theme = useTheme();
+	const onBridgeSuccess = useCallback(() => {
+		dispatch(fetchBalances());
+	}, [dispatch]);
+
+	const background = hexToRGB(theme.colors.selectedTheme.input.secondary.background);
+	const text = hexToRGB(theme.colors.selectedTheme.button.text.primary);
+	const primaryButtonBg = hexToRGB(theme.colors.selectedTheme.socket.accent);
+
+	const customize: SocketCustomizationProps = {
```

Comment:
Maybe we can make a `utils/socket.ts` file, with a function that takes the theme object and returns the customization object.

Reply:
Good idea. I can also move all the constants to `utils/socket.ts`.